selfdrive/car/subaru/carcontroller.py

```diff
@@ -1,30 +1,34 @@
-#from common.numpy_fast import clip
 from selfdrive.car import apply_std_steer_torque_limits
 from selfdrive.car.subaru import subarucan
-from selfdrive.car.subaru.values import DBC
+from selfdrive.car.subaru.values import DBC, PREGLOBAL_CARS
 from opendbc.can.packer import CANPacker
 
 
 class CarControllerParams():
-  def __init__(self):
-    self.STEER_MAX = 2047              # max_steer 4095
-    self.STEER_STEP = 2                # how often we update the steer cmd
-    self.STEER_DELTA_UP = 50           # torque increase per refresh, 0.8s to max
-    self.STEER_DELTA_DOWN = 70         # torque decrease per refresh
-    self.STEER_DRIVER_ALLOWANCE = 60   # allowed driver torque before start limiting
-    self.STEER_DRIVER_MULTIPLIER = 10  # weight driver torque heavily
-    self.STEER_DRIVER_FACTOR = 1       # from dbc
-
+  def __init__(self, car_fingerprint):
+    self.STEER_MAX = 2047                # max_steer 2047
+    self.STEER_STEP = 2                  # how often we update the steer cmd
+    self.STEER_DELTA_UP = 50             # torque increase per refresh, 0.8s to max
+    self.STEER_DELTA_DOWN = 70           # torque decrease per refresh
+    if car_fingerprint in PREGLOBAL_CARS:
+      self.STEER_DRIVER_ALLOWANCE = 75   # allowed driver torque before start limiting
```

Comment:
why is this different for pre-global cars? seems like they share the same EPS

Reply:
yes, looks like they can be unified to use Global's 60, since both scales are similar. I will change it

Reply:
Done